lib/generators/shopify_app/hmac_controller/templates/hmac_verification_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class HmacVerificationController < ApplicationController
```

Comment:
Can this controller just be part of the Gem rather than generated? It doesn't seem like something that the user customizes, just something that the use extends. If we are clever about this as well it can be reused for our webhook controller as well.

Reply:
I hid it from the namespace so that we can call it like a generator from other generators, though it won't come up in help and whatnot.